gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -80,10 +83,13 @@
 public class CommandTree extends Composite
     implements Tab, Capture.Listener, CommandStream.Listener {
   protected static final Logger LOG = Logger.getLogger(CommandTree.class.getName());
+  private static final String COMMAND_INDEX_HOVER = "Double click to copy index. Use Ctrl+G to do command jumping.";
+  private static final String COMMAND_INDEX_DSCRP = "Command index: ";
```

Comment:
"Command jumping" may not be the most understandable to our users, maybe it would be more understandable with something like: "Use Ctrl+G to jump to a given command index." ?

what do you think?

Reply:
Thanks! That sounds good, updated in a new commit. :)